### DISCUSSION TITLE
What former ☭ Republic name can be found at 🇺🇸?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1014] [Galaxy Trivia] [answered]
**Bonus:** How many European capitals start with the letter L?

**Comment by peterbaccaro** [comment]
During the Soviet era, which lasted from 1922 to 1991, Georgia (officially known as the **Republic of Georgia**) was one of the 15 constituent republics of the Soviet Union. The name of the former Soviet republic (i.e. Georgia) is also the name of a US state.  The US state of **Georgia** is located in the southeastern part of the country.

There are **four** European capitals start with the letter L, they are:
- Lisbon - The capital of Portugal.
- Ljubljana - The capital of Slovenia.
- London - The capital of England and the United Kingdom.
- Luxembourg (city) - The capital of Luxembourg.

**Comment by ParanoidUser** [comment]
**Extra:** Which country is home to the world's longest fence?

**Reply by peterbaccaro** [reply]
The world's longest fence is located in **Australia**. It is known as the **Dingo Fence** or **Dog Fence**, and it stretches for approximately 5,614 kilometers (3,488 miles) across the country. The fence was originally built to protect agricultural lands in southeastern Australia from the movement of dingoes, which are wild dogs found in the region. The Dingo Fence **starts near the eastern coast of Queensland and runs all the way to the cliffs of the Nullarbor Plain in South Australia**.

**Comment by thuongtruong1009** [comment, marked as answer]
The former Soviet Republic name that can be found in the United States is "Georgia." While Georgia is now an independent country in the Caucasus region, there is also a state named Georgia in the United States.

There is one European capital that starts with the letter L, which is "Lisbon." Lisbon is the capital of Portugal.
